src/components/safe-apps/SafeAppLandingPage/index.tsx

```diff
@@ -57,10 +57,6 @@ const SafeAppLanding = ({ appUrl, chain }: Props) => {
     )
   }
 
-  if (!safeApp) {
-    return <div>No Safe App found</div>
-  }
```

Comment:
Porque?

Reply:
`safeApp` can not be `null` / `undefined` because we always return `getEmptySafeApp` inside of `useSafeAppFromManifest`